### ISSUE TITLE
ERROR couldn't connect to zsys daemon

### ISSUE THREAD

**Issue by leoheck** [ubuntu/zsys#225]
I am using pre-released version of Ubuntu 22.04 and I am experiencing this issue for a couple of days.

```bash 
ERROR couldn't connect to zsys daemon: connection error: desc = "transport: Error while dialing dial unix /run/zsysd.sock: connect: connection refused" 
```

This is the first time using ZSH, and I don't know much about it.
Should I be worried about this message? 
Do you know any way to fix this?

Thank you.

**Comment by braydofficial** [comment]
Can confirm this issue. Have the same error on Ubuntu 22.04 Beta and didn't figure out the issue yet, too.

**Comment by YamiYukiSenpai** [comment]
same here

**Comment by jibel** [comment]
Thank you for your report. This is [bug 1967585](https://bugs.launchpad.net/ubuntu/+source/zsys/+bug/1967585) . We are looking into it.

**Comment by leoheck** [comment]
After the upgrade today, the meesage changed a bit. 
```
ERROR couldn't connect to zsys daemon: connection closed 
```

**Comment by jibel** [comment]
@leoheck yes this is the same issue, the daemon cannot start. So while we are looking into it and until it is fixed you can downgrade and pin 0.5.8.
It can be downloaded from https://launchpad.net/ubuntu/+source/zsys/0.5.8/+build/21362897/+files/zsys_0.5.8_amd64.deb

**Comment by leoheck** [comment]
Seems to be a good workaround. Thanks.

**Comment by leoheck** [comment]
@jibel how can I know if and when this issue is fixed by having the 0.5.8 installed and pinned? Normal upgrades are going to tell me something?

**Comment by jibel** [comment]
This is fixed in zsys 0.5.9 which is being released to the distro.

**Comment by leoheck** [comment]
I could not see any improvement in this bug yet.
```
ERROR couldn't connect to zsys daemon: connection error: desc = "transport: Error while dialing dial unix /run/zsysd.sock: connect: connection refused" 
```

```
➜  ~ sudo apt-cache policy zsys
zsys:
  Installed: 0.5.9
  Candidate: 0.5.9
  Version table:
 *** 0.5.9 500
        500 http://archive.ubuntu.com/ubuntu jammy/main amd64 Packages
        100 /var/lib/dpkg/status
➜  ~ 
```

How do I open this ticket again?
